OneCommunityGlobal/HGNRest
# Imran split blue squares permissions

# Description
handles create, edit and delete blue square permission to any user.

## Related PRS (if any):
To test this backend PR you need to checkout the frontend [PR#2248](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2248).
…

## Main changes explained:
- added addInfringements, editInfringement and deleteInfringement permission to the list
- added check for permission before creating or editing or deleting blue square in profile controller
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. follow the steps on the frontend PR to complete testing
